server/controllers/userController.js

```diff
@@ -89,30 +89,30 @@ const register = async (req, res) => {
     return res
       .cookie('token', token, { httpOnly: true })
       .send({ data: newUser.rows[0] });
-  } catch (error) {
-    // DB returned a error
-    // We might not want to pass the error directly to the front end...
-    // ...anybody have any other ideas how to handle this?
-    return res.json({ error });
+  } catch (err) {
```

Comment:
Check out server/middleware/errorHandling.js

The way this is written you'll overwrite the message and serverMessage properties in the default ErrObj. What we really want is for the serverMessage to log the error object you're catching here. ie return next(err) SHOULD do the trick. It is ok to overwrite the message if you want, but the serverMessage should be err rather than a new object with an err property.